app/js/controllers/foyer/pensionsAlimentaires.js

```diff
@@ -24,13 +24,13 @@ angular.module('ddsApp').controller('FoyerPensionsAlimentairesCtrl', function($s
             })
             .value();
 
-        return result || 0;
+        return Math.round(result) || 0;
```

Comment:
It could be, and this was a broad behaviour of mes-aides. Changing it to rounding to cents.